src/service-object.ts

```diff
@@ -409,11 +415,11 @@ class ServiceObject extends EventEmitter {
 
     reqOpts.interceptors_ = childInterceptors.concat(localInterceptors);
 
-    if (!callback) {
+    if (returnStream) {
```

Comment:
Just a variable naming preference you are free to disregard. `shouldReturnStream` indicates to me it's a boolean, where as `returnStream` makes me think it's a Stream that we're returning.